packages/core/src/builder.class.ts

```diff
@@ -118,7 +117,6 @@ function setCookie(name: string, value: string, expires?: Date) {
       (value || '') +
       expiresString +
       '; path=/' +
-      `; domain=${getTopLevelDomain(location.hostname)}` +
```

Comment:
is this not a significant change? we're no longer setting `domain` on all of our cookies. surprised to see as part of the personalization feature.